benchmarks/1-vu.script.js

```diff
@@ -0,0 +1,25 @@
+import http from "k6/http";
+import { check, sleep } from "k6";
+import { describe } from 'https://jslib.k6.io/expect/0.0.4/index.js';
```

Comment:
Can we avoid importing from remote https and import from local file?

Reply:
Wow, didn't know this could be done in ts